monitoring/README.md

```diff
@@ -17,3 +17,62 @@ task metrics.
 
 To load this dashboards in Grafana, you need at least one public agent on your
 cluster to access to the Grafana UI. Grafana is exposed in the url: `http://<public_agent_ip>:<grafana_port>`.
+
+## Backing up dashboards
+
+A tool like [`grafcli`](https://github.com/m110/grafcli) can be used to backup the dashboards on a Grafana server.
+
+### Install `grafcli` using pip:
```

Comment:
Does this get run on the host that is running Grafana?

Reply:
Oh, I see, looks like you run this anywhere and the config file points to the Grafana host?

Reply:
Yes, this is installed locally and communicates with the Grafana host over the API.